c7n/resources/lex.py

```diff
@@ -40,6 +42,90 @@ class resource_type(query.TypeInfo):
     source_mapping = {"describe": query.DescribeWithResourceTags, "config": query.ConfigSource}
 
 
+class LexV2BotAliasDescribe(query.ChildDescribeSource):
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('lexv2-models')
+        for r in resources:
+            botalias = client.describe_bot_alias(
+                botId=r['c7n:parent-id'], botAliasId=r['botAliasId'])
+            r.update(botalias)
+        return universal_augment(self.manager, resources)
+
+
+@resources.register('lexv2-bot-alias')
+class LexV2BotAlias(query.ChildResourceManager):
+    class resource_type(query.TypeInfo):
+        service = 'lexv2-models'
+        parent_spec = ('lexv2-bot', 'botId', True)
+        enum_spec = ('list_bot_aliases', 'botAliasSummaries', None)
```

Comment:
Actually, the above mentioned comment re: applying filter on botAliasStatus can be done here

Reply:
Nvm, looks like this has to be on the client side